db/seeds.rb

```diff
@@ -5,3 +5,13 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'csv'
+CSV.foreach(Rails.root.join("db/seeds_data/locationseed.csv"), headers: true) do |row|
```

Comment:
I don't think we want to set the :headers option to true -- it seems to be ignoring the first row